.eslintrc.js

```diff
@@ -65,6 +103,10 @@ module.exports = {
     'no-param-reassign': 'off',
     radix: 'off',
     'require-atomic-updates': 'off',
+    'jsdoc/match-description': [
```

Comment:
Another thing that's being added temporarily to allow for a list of links at the end of a description block. Will also move this to a separate PR, but here for now to get the linter step to pass.